totalopenstation/formats/__init__.py

```diff
@@ -56,30 +56,27 @@ class FeatureCollection(g.FeatureCollection):
     pass
 
 
-class Parser:
+class Parser(object):
```

Comment:
I don't think this is necessary since all classes are new style classes in Python 3

Reply:
Yes, these changes were made before py3 migration ^^
I drop some py2-py3 changes on these commits but not all.
Will revert those.